### PR TITLE
add getBounds, getUsableBounds, getNumberOfDisplays, getDisplays to Display

### PR DESCRIPTION
**Pull request by glennsl**
I'm trying to figure out a reliable way to get the center point, and am thinking that getting the bounds of each display is a good first step at least. Unfortunately this crashes when running the example, and I don't understand why. This seems to be doing exactly what the display mode functions right above are doing, just with four fields instead of three.

### CODE REVIEW

File: src/sdl2_wrapper.cpp [hunk 1]
```diff
@@ -459,6 +459,40 @@ CAMLprim value resdl_SDL_GetDesktopDisplayMode(value vDisplay) {
...
+
+  int displayIndex = Int_val(vDisplay);
+  SDL_Rect sdlRect;
+
+  SDL_GetDisplayUsableBounds(displayIndex, &sdlRect);
+
+  rect = caml_alloc(4, 0);
+  Store_field(rect, 0, sdlRect.x);
+  Store_field(rect, 1, sdlRect.y);
+  Store_field(rect, 2, sdlRect.h);
```
Comment (bryphe) [inline src/sdl2_wrapper.cpp]: Noticed one issue - `sdlRect.h` is here twice

File: src/sdl2_wrapper.cpp [hunk 2]
```diff
@@ -459,6 +459,40 @@ CAMLprim value resdl_SDL_GetDesktopDisplayMode(value vDisplay) {
...
+  CAMLparam1(vDisplay);
+  CAMLlocal1(rect);
+
+  int displayIndex = Int_val(vDisplay);
+  SDL_Rect sdlRect;
+
+  SDL_GetDisplayBounds(displayIndex, &sdlRect);
+
+  rect = caml_alloc(4, 0);
+  Store_field(rect, 0, sdlRect.x);
```
Comment (bryphe) [inline src/sdl2_wrapper.cpp]: These should be `Store_field(rect, 0, Val_int(sdlRect.x))`, I believe

File: src/sdl2_wrapper.cpp (hunk 1 above)
Comment (bryphe) [reply src/sdl2_wrapper.cpp]: This one should be `sdlRect.w`

File: src/sdl2_wrapper.cpp (hunk 2 above)
Comment (bryphe) [reply src/sdl2_wrapper.cpp]: Changing all of those seems to fix the crash 👍

File: src/sdl2_wrapper.cpp (hunk 1 above)
Comment (glennsl) [reply src/sdl2_wrapper.cpp]: Hmm, I was sure I fixed that. Good catch though!

File: src/sdl2_wrapper.cpp (hunk 2 above)
Comment (glennsl) [reply src/sdl2_wrapper.cpp]: It isn't with the display mode functions above. But still certainly worth a try.

File: src/sdl2_wrapper.cpp (hunk 1 above)
Comment (glennsl) [reply src/sdl2_wrapper.cpp]: Oh, I fixed one function but not the other :facepalm:

File: src/sdl2_wrapper.cpp (hunk 2 above)
Comment (glennsl) [reply src/sdl2_wrapper.cpp]: Yep, that was it. Thanks! :tada:

**Review by bryphe — APPROVED** [review]
Looks great!

**Pull request merged**